xwiki-mysql-tomcat/Dockerfile

```diff
@@ -32,12 +32,18 @@ ENV XWIKI_VERSION=8.4.4 \
 RUN echo "deb http://http.debian.net/debian jessie-backports main" > /etc/apt/sources.list.d/backports.list && \
   apt-get update && \
   apt-get -y upgrade && \
-  apt-get -y --force-yes install sudo nano unzip curl openjdk-8-jdk tomcat8 libreoffice
+  apt-get --no-install-recommends -y --force-yes install sudo nano unzip curl openjdk-8-jdk tomcat8 libreoffice && \
```

Comment:
hmm this is strange, I had committed a change to have each package on a new line and I don't see it in this diff. See https://github.com/xwiki-contrib/docker-xwiki/blob/master/xwiki-mysql-tomcat/Dockerfile#L35

Any idea?